src/Chapter13.Tests/Listing13.15.Tests.cs

```diff
@@ -0,0 +1,30 @@
+﻿
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter13.Listing13_15.Tests;
+
+[TestClass]
+public class ProgramTests
+{
+    [TestMethod]
+    public void MainTest()
```

Comment:
This is great, but we have a built in tool for doing console assertions that can make this a bit easier. If you look [here](https://github.com/IntelliTect/EssentialCSharp/blob/v12.0/src/Chapter13.Tests/Listing13.12.Tests.cs) (other tests as well), you can see the syntax for leveraging the ConsoleAssert class to be able to test the input/output from the console.

Reply:
I saw that all of the tests in 13 were similar to that, but when I tried implementing it, I wasn't able to get it to pass without implementing my own version, but I'll try to get it to pass with the built in tool